plugins/format-api-f14b86/package.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "format-api-f14b86",
```

Comment:
As per the [conventions](https://github.com/wptrainingteam/wp-block-development-examples/blob/trunk/CONTRIBUTIONS.md#conventions-used-for-the-examples-in-this-repo) 
-   On **`<plugin-folder>/package.json`**
    -   [ ] **`name`** should follow the convention `@wp-block-development-examples/<slug>`
        -   Ex: `"name": "@wp-block-development-examples/block-static-rendering-b16608"`